do not launch tracker on startup 
not suitable for upstream
diff -r 34c39783c8b7 src/miners/fs/tracker-miner-fs.desktop.in.in
--- a/src/miners/fs/tracker-miner-fs.desktop.in.in	Thu Oct 01 13:39:31 2020 +0200
+++ b/src/miners/fs/tracker-miner-fs.desktop.in.in	Thu Oct 01 13:42:31 2020 +0200
@@ -3,6 +3,7 @@
 _Comment=Crawls and processes files on the file system
 Icon=
 Exec=@libexecdir@/tracker-miner-fs
+Hidden=true
 Terminal=false
 Type=Application
 Categories=Utility;
diff -r 34c39783c8b7 src/miners/rss/tracker-miner-rss.desktop.in.in
--- a/src/miners/rss/tracker-miner-rss.desktop.in.in	Thu Oct 01 13:39:31 2020 +0200
+++ b/src/miners/rss/tracker-miner-rss.desktop.in.in	Thu Oct 01 13:42:31 2020 +0200
@@ -3,6 +3,7 @@
 _Comment=Fetch RSS/ATOM feeds
 Icon=
 Exec=@libexecdir@/tracker-miner-rss
+Hidden=true
 Terminal=false
 Type=Application
 Categories=Utility;
diff -r 34c39783c8b7 src/tracker-extract/tracker-extract.desktop.in.in
--- a/src/tracker-extract/tracker-extract.desktop.in.in	Thu Oct 01 13:39:31 2020 +0200
+++ b/src/tracker-extract/tracker-extract.desktop.in.in	Thu Oct 01 13:42:31 2020 +0200
@@ -3,6 +3,7 @@
 _Comment=Extracts metadata from local files
 Icon=
 Exec=@libexecdir@/tracker-extract
+Hidden=true
 Terminal=false
 Type=Application
 Categories=Utility;
